ci-operator/jobs/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master-postsubmits.yaml

```diff
@@ -4,12 +4,12 @@ postsubmits:
     always_run: true
     branches:
     - ^master$
-    cluster: build09
+    cluster: build10
     decorate: true
     decoration_config:
       skip_cloning: true
     labels:
-      ci-operator.openshift.io/cluster: build09
+      ci-operator.openshift.io/cluster: build10
```

Comment:
this isn't related to your PR is it? I see a lot of the build cluster changes in this PR for jobs that aren't related to kubevirt

Reply:
I ran a `make update` and it added all this stuff.